jdbc/src/main/java/org/apache/zeppelin/jdbc/SqlCompleter.java

```diff
@@ -179,7 +179,8 @@ public int complete(String buffer, int cursor, List<InterpreterCompletion> candi
 
 
   private static void fillTableNames(String schema, DatabaseMetaData meta, Set<String> tables) {
-    try (ResultSet tbls = meta.getTables(schema, schema, "%", null)) {
+    try (ResultSet tbls = meta.getTables(schema, schema, "%",
+        new String[]{"TABLE", "VIEW", "ALIAS", "SYNONYM", "GLOBAL TEMPORARY", "LOCAL TEMPORARY"})) {
```

Comment:
"SYNONYM", "GLOBAL TEMPORARY", "LOCAL TEMPORARY" might not be globally applicable to all JDBC sources